sklearn/preprocessing/data.py

```diff
@@ -2873,3 +2873,39 @@ def inverse_transform(self, X):
                     X_tr[mask, idx] = None
 
         return X_tr
+
+def get_feature_names(self, input_features=None):
+        """
+        Return feature names for output features
+
+        Parameters
+        ----------
+        input_features : list of string, length n_features, optional
+            String names for input features if available. By default,
+            "x0", "x1", ... "xn_features" is used.
+
+        Returns
+        -------
+        output_feature_names : list of string, length n_output_features
+
+        """
+        cats = self.categories_
+        feature_names = []
+        if input_features is None:
+            input_features = ['x%d' % i for i in range(len(cats))]
+
+        cats2=[]
+        for i in range(len(cats)):
+            temp=[]
+            for t in cats[i]:
+                temp.append(str(t))
+            cats2.append(np.array(temp))
+
+   	
```

Comment:
This line has spaces in it, which should be removed. It also has a tab character.